src/components/AvatarMenu/AvatarMenu.styles.ts

```diff
@@ -0,0 +1,51 @@
+import { styled } from '@mui/material/styles';
+import { IconButton, Typography } from '@mui/material';
+import type { PropsStyledAvatarMenu } from './AvatarMenu.types';
+import { theme } from '../../style-library';
+
+export const AvatarMenuRoot = styled(IconButton, {
+	shouldForwardProp: (prop) => prop !== '$fullWidth',
+})<PropsStyledAvatarMenu>((props) => {
```

Comment:
shouldForwardProp excludes only '$fullWidth' but not '$isOpen'. In AvatarMenu you pass $isOpen to AvatarMenuRoot (see AvatarMenu.tsx line 43). Because $isOpen is not filtered it will be forwarded to the DOM producing React warnings for unknown attributes.

**Sugerencia:**
```typescript
Exclude all $-prefixed styling-only props from being forwarded. Example:
shouldForwardProp: (prop) => prop !== '$fullWidth' && prop !== '$isOpen'
```